test/m4/appframework/appframework_test.go

```diff
@@ -154,6 +155,39 @@ var _ = Describe("m4appfw test", func() {
 
 			//Verify Apps are installed cluster-wide
 			testenv.VerifyAppInstalled(deployment, testenvInstance, testenvInstance.GetName(), allPodNames, appListV2, true, "enabled", true, true)
+
+			// Get instance of current Indexer CR with latest config
+			idxcName := deployment.GetName() + "-" + "site1"
+			idxc := &enterprisev1.IndexerCluster{}
+			err = deployment.GetInstance(idxcName, idxc)
+			Expect(err).To(Succeed(), "Failed to get instance of Indexer Cluster")
+			defaultIndexerReplicas := idxc.Spec.Replicas
+			scaledIndexerReplicas := defaultIndexerReplicas + 1
+			testenvInstance.Log.Info("Scaling up Indexer Cluster", "Current Replicas", defaultIndexerReplicas, "New Replicas", scaledIndexerReplicas)
+
+			// Update Replicas of Indexer Cluster
+			idxc.Spec.Replicas = int32(scaledIndexerReplicas)
+			err = deployment.UpdateCR(idxc)
+			Expect(err).To(Succeed(), "Failed to Scale Up Indexer Cluster")
+
+			// Ensure Indxer cluster scales up and go to ScalingUp phase
+			testenv.VerifyIndexerClusterPhase(deployment, testenvInstance, splcommon.PhaseScalingUp, idxcName)
+
+			// Ensure Indexer cluster go to Ready phase
+			testenv.IndexersReady(deployment, testenvInstance, siteCount)
+
+			// Verify RF SF is met
+			testenv.VerifyRFSFMet(deployment, testenvInstance)
```

Comment:
nit: do we need to test this?

Reply:
This is make sure the cluster scale up did go as expected.

Reply:
I am not sure if scaling up has anything to do with RF/SF being met. You are already checking the indexers ready phase prior to this.

Reply:
It's part of checks to make cluster is in stable state. I get your point about scaling should ideally not have anything to do with it, in that case this check will not effect the test case